notebooks/SITCOMTN_131_M1M3_VMS_TMA_Coupling_investigation.ipynb

```diff
@@ -0,0 +1,1197 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    ax[1].plot(df_ele.index, df_ele["actualPosition"], label="Declination")
</pre><p>Declination --&gt; Elevation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-131/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>Argh ! I thought that I had corrected all of these. Thanks !</p>